contrib/compose/README.md

```diff
@@ -0,0 +1,41 @@
+# Running dex in Docker compose
+
+This demo intends to demonstrate Dex on a development workstation. It is only intended for local test setups and should not be used for production environments. I recommend to use k8s in production.
```

Comment:
Nit: Avoid terms such as "I" and "you"

"I recommend to use k8s in production." vs. "Kubernetes is recommended for production uses."

"You will see `sh -c 'sleepx';` to enforce the right order" vs, "Sleep statements are used to enforce the correct order."